tradeexecutor/ethereum/uniswap_v2/uniswap_v2_live_pricing.py

```diff
@@ -63,6 +63,8 @@ def __init__(self,
 
         self.uniswap_cache: Dict[TradingPairIdentifier, UniswapV2Deployment] = {}
 
+        self.epsilon = 1e-10
```

Comment:
Do not use hardcode value - if you need epsilon there are two options in `account_correction` module.

Also make it so that you can pass this as a parameter, so it can be overridden in unit tests, making writing epsilon tests easier.